keys/keys.go

```diff
@@ -121,9 +121,15 @@ func RangeStatsKey(rangeID roachpb.RangeID) roachpb.Key {
 
 // SequenceCacheKey returns a range-local key by Range ID for a
 // sequence cache entry, with detail specified by encoding the
-// supplied client command ID.
-func SequenceCacheKey(rangeID roachpb.RangeID, family []byte) roachpb.Key {
-	return MakeRangeIDKey(rangeID, LocalSequenceCacheSuffix, encoding.EncodeBytes(nil, family))
+// supplied transaction ID and sequence number.
+// The length of the prefix of the encoded key excluding the encoded
+// sequence number is returned.
+func SequenceCacheKey(rangeID roachpb.RangeID, id []byte, seq uint32) (roachpb.Key, int) {
```

Comment:
Instead of a second return value which is thrown away at nearly every call site, the prefix could be computed via a second method which doesn't take the sequence number (see RaftLogKey/RaftLogPrefix)